docs/quickstart.rst

```diff
@@ -158,15 +189,16 @@ Access model updates
    In FEDn Studio, you can access global model updates by going to the 'Models' or 'Sessions' tab. Here you can download model updates, metrics (as csv) and view the model trail.
 
 
-You can access global model updates via the APIClient:
+You can also access global model updates via the APIClient:
 
 .. code:: python
 
    >>> ...
    >>> client.download_model("<model-id>", path="model.npz")
 
 
-**Connecting clients using Docker**
+Connecting clients using Docker
```

Comment:
Are we sure this is needed in the "Getting started" section?

Reply:
Fixed.